test/integration/font-optimization/test/index.test.js

```diff
@@ -250,50 +234,6 @@ describe('Font Optimization', () => {
         runTests()
       })
 
-      describe('Font optimization for serverless apps', () => {
```

Comment:
Serverless tests are failing. After discussing it with JJ, I'm removing the tests since they are going to be deprecated .